Sources/Property.swift

```diff
@@ -678,3 +759,73 @@ public final class MutableProperty<Value>: MutablePropertyProtocol {
 		observer.sendCompleted()
 	}
 }
+
+extension MutableProperty {
```

Comment:
Perhaps this can wait until `ComposableMutablePropertyProtocol` in #182 lands. hmm?